projects/sandbox/deploy/deploy/main.py

```diff
@@ -1,25 +1,59 @@
 import logging
 import time
+from dataclasses import asdict, dataclass
 from pathlib import Path
-from typing import List
+from typing import List, Literal
 
 import numpy as np
 import torch
 from deploy.dataloading import data_iterator
+from ligo.gracedb.rest import GraceDb
 
 from bbhnet.analysis.ledger.events import EventSet
 from bbhnet.architectures import ResNet
 from bbhnet.logging import configure_logger
 
+Gdb_server = Literal["playground", "test", "production"]
+
 
 def unfold(x, kernel_size: int, stride: int):
     # TODO: fill out
     return
 
 
-def submit_trigger():
-    # TODO: fill out
-    return
+def connect_to_gracedb(server: Gdb_server):
+    if server in ["playground", "test"]:
+        server = f"https://gracedb-{server}.ligo.org/api/"
+    elif server == "production":
+        server = "https://gracedb.ligo.org/api/"
+    else:
+        raise ValueError(f"Unknown server {server}")
+
+    gdb = GraceDb(service_url=server)
```

Comment:
Do we need to provide credentials here?

Reply:
I think we can provide them explicitly, but it should also pick them up automatically via environment variables (see https://ligo-gracedb.readthedocs.io/en/latest/api.html). However we do need GraceDB folks to authorize specific users (i.e Will Ethan Rafia Alec etc.) to upload a "BBHnet" (or whatever we end up calling it) event. 

Down the line, we'll likely make a ligo "BBHnet" account from which we will run production analysis and submit triggers.